### ISSUE TITLE
--fix adds trailing space to empty comment lines, then fails yaml[trailing-spaces]

### ISSUE THREAD

**Issue by colinbrophy-genius** [ansible/ansible-lint#4902]
When running `ansible-lint --fix` on files containing empty comment lines (just `#` used as section separators), the fixer rewrites them as `# ` (hash + space), which then triggers a `yaml[trailing-spaces]` violation that cannot be auto-fixed.

## Steps to reproduce

1. Create a YAML file with an empty comment line:
```yaml
---
# Header comment
#
# More comments
- name: Test task
  debug:
    msg: hello
```

2. Run `ansible-lint --fix`

3. The `#` line becomes `# ` (trailing space added)

4. `yaml[trailing-spaces]` violation is reported with "Rule specific fix not applied"

**Using ansible-lint version: 25.12.1**

**Comment by emmanuel-ferdman** [comment]
Hi @colinbrophy-genius this issue was fixed in PR #4868, which was included in version 25.12.2. You mentioned that you are using 25.12.1, which predates the fix. Could you upgrade to one of the latest versions and test it again?

**Comment by colinbrophy-genius** [comment]
Working now, thank you. Apologies for missing the other issue raised about this and not testing on latest.
